apps/web/playwright/organization/booking-list.e2e.ts

```diff
@@ -0,0 +1,374 @@
+      await assertBookingVisibleFor(team1_teammate1, page, eventType, true);
+    });
+
+    test("Round Robin EventType booking list", async ({ page, users, orgs }) => {
+      const { org, orgOwner, orgAdmin, team1owner, team2owner, commonUser, team1_teammate1 } =
+        await createOrg(orgs, users);
+
+      const { team: team1 } = await team1owner.getFirstTeamMembership();
+      const scenario = {
+        schedulingType: SchedulingType.ROUND_ROBIN,
+        teamEventTitle: `round-robin-team-event`,
+        teamEventSlug: slugify(`round-robin-team-event-${randomString(5)}`),
+      };
+
+      const eventType = await createTeamEventType(team1owner, team1, scenario);
+      const { id: eventId } = eventType;
+      await prisma.host.createMany({
+        data: [
+          {
+            userId: commonUser.id,
+            eventTypeId: eventId,
+          },
+          {
+            userId: team1_teammate1.id,
+            eventTypeId: eventId,
+          },
+        ],
+      });
+
+      await prisma.host.deleteMany({
+        where: {
+          userId: team1owner.id,
+          eventTypeId: eventId,
+        },
+      });
+
+      await expectPageToBeNotFound({ page, url: `/team/${team1.slug}/${eventType.slug}` });
+
+      await doOnOrgDomain(
+        {
+          orgSlug: org.slug,
+          page,
+        },
+        async () => {
+          await bookTeamEvent({ page, team: team1, event: eventType });
+        }
+      );
+      // booking should be visible for the Org OWNER even though he is not part of the booking
+      await assertBookingVisibleFor(orgOwner, page, eventType, true);
+      // booking should be visible for the Org ADMIN even though he is not part of the booking
+      await assertBookingVisibleFor(orgAdmin, page, eventType, true);
+      // booking should be visible for the team host even though he is not part of the booking
+      await assertBookingVisibleFor(team1owner, page, eventType, true);
+      // booking should not be visible for other team host
+      await assertBookingVisibleFor(team2owner, page, eventType, false);
+    });
+
+    test("Managed EventType booking list", async ({ page, users, orgs }) => {
+      const { org, orgOwner, orgAdmin, team1owner, team2owner, commonUser, team1_teammate1 } =
+        await createOrg(orgs, users);
+
+      const { team: team1 } = await team1owner.getFirstTeamMembership();
+
+      await team1owner.apiLogin();
+      await page.goto(`/event-types?dialog=new&eventPage=team%2F${team1.slug}&teamId=${team1.id}`);
+      await page.getByTestId("managed-event-type").click();
+      await page.getByTestId("event-type-quick-chat").click();
+      await page.getByTestId("event-type-quick-chat").fill("managed-event");
+      await page.getByRole("button", { name: "Continue" }).click();
+      await page.getByTestId("vertical-tab-assignment").click();
+
+      await page.locator("span:nth-child(3) > .bg-default > div > .text-emphasis").nth(0).click();
```

Comment:
Please try to use data testid or something else here as this would easily break when ui is changed.

Reply:
we dont really need it, test is working fine without it 
removed